From 77f8cea06ca22bf0ea5593fa17dda12a7df7a4f1 Mon Sep 17 00:00:00 2001
From: Kamran Ali <kamranali.says@gmail.com>
Date: Mon, 16 Dec 2024 07:49:24 +0000
Subject: [PATCH] Create README.md

---
 README.md | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..b210307
--- /dev/null
+++ b/README.md
@@ -0,0 +1,8 @@
+# Open URL in chrome by node js code
+
+# Install following modules
+npm install puppeteer
+
+# Run Application
+node filename
+
-- 
2.46.2.windows.1

